docs/release_notes.rst

```diff
@@ -4,6 +4,14 @@ Release Notes
 **pydocstyle** version numbers follow the
 `Semantic Versioning <http://semver.org/>`_ specification.
 
+
+Current development version
+---------------------------
+
+Bug Fixes
+
+* Add readthedocs configuration to fix GitHub action (#657).
```

Comment:
This isn't a bug fix that affects users in any meaningful way. We don't need it in the changelog